src/Mvc/Mvc.Core/src/Internal/MvcCoreLoggerExtensions.cs

```diff
@@ -173,10 +173,10 @@ static MvcCoreLoggerExtensions()
                 1,
                 "Executing action method {ActionName} - Validation state: {ValidationState}");
 
-            _actionMethodExecutingWithArguments = LoggerMessage.Define<string, string[], ModelValidationState>(
-                LogLevel.Information,
-                1,
-                "Executing action method {ActionName} with arguments ({Arguments}) - Validation state: {ValidationState}");
+            _actionMethodExecutingWithArguments = LoggerMessage.Define<string, string[]>(
+                LogLevel.Debug,
```

Comment:
This has to be `Trace` not debug. Debug is not supposed to include sensitive information.

Reply:
I did consider setting this to trace. While not identical, we do log some other things including the RVD at Debug level: https://github.com/aspnet/AspNetCore/blob/ee0fb6d0e268afcb4848c64ef42a0cb0f48f8e3f/src/Mvc/Mvc.Core/src/Internal/MvcCoreLoggerExtensions.cs#L779-L782

Should I do a pass over these?

Reply:
We don't consider route values to be sensitive.